plugins/git/git.plugin.zsh

```diff
@@ -49,7 +49,7 @@ function work_in_progress() {
 alias g='git'
 
 alias ga='git add'
-
+alias gaa='git add --all'
```

Comment:
The newline was probably meant to separate `git add ..` from `git branch ..`.
But there's no need to update the PR probably, as it will be added in the hotfix-PR probably.